.travis.yml

```diff
@@ -0,0 +1,105 @@
+stages:
+  - name: self-test
+  - name: integration
+    if: branch = master OR type = pull_request OR branch = dev_ops/travis
+
+_integration_job_template: &integration_job_template
+  stage: integration
+  cache: npm
+  before_install:
+    - nvm install 11
+    - node --version
+  install:
+    - gem install xcodeproj
+    - npm install
+    - npm install -g react-native-cli
+  before_script:
+    - cd integration-tests
+    - bash makeProject.sh $RN_VERSION
+    - cd Setup
+    - bash ../installFlow.sh
+    - rm -rf ../../node_modules
+
+_android_job_template: &android_job_template
+  <<: *integration_job_template
+  language: android
+  jdk: openjdk8
+  android:
+    components:
+      - tools
+      - platform-tools
+      - build-tools-23.0.1
+      - build-tools-26.0.3
+      - build-tools-27.0.3
+      - build-tools-28.0.3
+      - android-25
+      - android-26
+      - android-27
+      - android-28
+      - extra-android-m2repository
+      - extra-google-m2repository
+      - extra-android-support
+      - sys-img-x86-android-26
+    licenses:
+      - 'android-sdk-license-.+'
+  script:
+    - node_modules/.bin/flow check
```

Comment:
Flow is already run with: `npm run lint`?

Reply:
Nope, it `npm run lint` happens when there's no generated project yet. Since `flow-bin` version might differ for lib I wanted to run flow separately just for app.